finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -128,9 +128,16 @@ object HighResTimer {
    *
    * It is a shared resource and as such, `stop` is ignored.
    */
-  val Default: com.twitter.util.Timer =
+  private[finagle] val Default: com.twitter.util.Timer =
     new JavaTimer(true, Some("HighResTimer"))
```

Comment:
we should keep stop hidden, but we can make another method `private[HighResTimer] def hiddenStop(): Unit = super.stop()`, and then have stop use that method

Reply:
Such method will not be visible because it is not defined in com.twitter.util.Timer.
I have to create new class that extends com.twitter.util.Timer.
Do you still want to do it in that way?